### PR TITLE
Improve calculator

### PR DESCRIPTION
**Pull request by ValenMorales**

### CODE REVIEW

**Review by L-Zuluaga — CHANGES_REQUESTED** [review]
Most of the files lack of an end of line

**Comment by ValenMorales** [comment]
Does this look alright?

File: calculator/methods.rb
Comment (L-Zuluaga) [inline calculator/methods.rb:1]: This file still lacks an end of line

**Review by L-Zuluaga — APPROVED** [review]
Yes, however there is still a module without the end of line, it's a good practice to include one in all of the files you include on your code 👍🏻

**Pull request merged**